compose.yaml

```diff
@@ -8,10 +8,11 @@ services:
       - postgres
       - redis
     ports:
-      - "${REDASH_PORT:-5001}:5000"
+      - "${REDASH_PORT:-5001}:${REDASH_PORT:-5001}"
```

Comment:
Why the port number change here and below?

Reply:
it also happens in entrypoint, otherwise it will not work.

Reply:
Sure, I understand that.  I'm asking why it's in this PR? :smile: